### PR TITLE
change python solution for mixing milk

### PR DESCRIPTION
**Pull request by jjliewie**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which includes the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: content/2_Bronze/Simulation.mdx [hunk 1]
```diff
@@ -270,31 +270,32 @@ public class MixMilk {
...
 capacity[i] is the maximum capacity of bucket i
 milk[i] is the current amount of milk in bucket i
 """
 
-def pour(i, j):
-	amt = min(milk[i], capacity[j]-milk[j])
-	# Amount of milk to pour is the minimum of the remaining milk in bucket i
-	# and the available capacity in bucket j
+i = 0
+for _ in range(100):
```
[the remark this answers is not in the document]
Comment (SansPapyrus683) [reply content/2_Bronze/Simulation.mdx:284]: just put it in a variable like `TURN_NUM` or something

[the remark this answers is not in the document]
Comment (jjliewie) [reply content/2_Bronze/Simulation.mdx:284]: wait are you talking about i or _

[the remark this answers is not in the document]
Comment (SansPapyrus683) [reply content/2_Bronze/Simulation.mdx:284]: i was talking about the `100`

[the remark this answers is not in the document]
Comment (jjliewie) [reply content/2_Bronze/Simulation.mdx:284]: oh..

File: content/2_Bronze/Simulation.mdx [hunk 2]
```diff
@@ -281,7 +281,9 @@ milk[i] is the current amount of milk in bucket i
 """
 
 i = 0
-for _ in range(100):
```
Comment (SansPapyrus683) [inline content/2_Bronze/Simulation.mdx]: put at top and make all caps

**Review by SansPapyrus683 — APPROVED** [review]

**Review by ryanchou-dev — APPROVED** [review]
⭐

**Pull request merged**